soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -24,6 +24,10 @@ config ENTROPY_MCUX_TRNG
 	default y if HAS_MCUX_TRNG
 	depends on ENTROPY_GENERATOR
 
+config ENTROPY_MCUX_CAAM
```

Comment:
remove this, the driver will get default y if its enabled in devicetree so this isn't needed.